JitsiConference.js

```diff
@@ -2706,14 +2671,10 @@ JitsiConference.prototype.sendFeedback = function(overallFeedback, detailedFeedb
 };
 
 /**
- * Returns true if the callstats integration is enabled, otherwise returns
- * false.
- *
- * @returns true if the callstats integration is enabled, otherwise returns
- * false.
+ * @returns false, since callstats in not supported anymore.
  */
 JitsiConference.prototype.isCallstatsEnabled = function() {
-    return this.statistics.isCallstatsEnabled();
```

Comment:
Maybe we should add a deprecation notice in the js docs and add a print that this will be removed, wdyt?